src/components/YourCompany/NameTranslations/NameTranslation.vue

```diff
@@ -0,0 +1,360 @@
+<template>
+  <div id="name-translation">
+    <confirm-dialog
+      ref="confirmTranslationDialog"
+      attach="#name-translation"
+    />
+    <v-layout v-if="!isEditing">
+      <v-flex xs3>
+        <label><strong>Name Translation(s)</strong></label>
+        <v-flex md1>
+            <v-chip v-if="hasNameTranslationChange" x-small label color="#1669BB" text-color="white">
+                Corrected
+            </v-chip>
+          </v-flex>
+      </v-flex>
+      <v-flex xs7 v-if="draftTranslations && draftTranslations.filter(x => x.action !== 'removed').length">
+        <div v-for="(translation, index) in draftTranslations.filter(x => x.action !== 'removed')"
```

Comment:
The filter code here and in the line above would be better as a getter due to caching.